### DISCUSSION TITLE
Ricerca Nominativi

### DISCUSSION THREAD

**Discussion by BS240872** [ondata/italian-public-sector-pnrr-data-guide#28] [Richiesta di aiuto]
### Testo

Non riesco ad acquisire esiti di gara per nominativo e per localizzazione , nominativi e localizzazione non sono mai associati

**Comment by aborruso** [comment]
> Non riesco ad acquisire esiti di gara per nominativo e per localizzazione , nominativi e localizzazione non sono mai associati

Ciao @BS240872 scusaci, ma il tuo messaggio ci era sfuggito. È passato molto tempo: hai risolto?

Se non hai risolto, mi fai un esempio dell'obiettivo che hai e che non riesci a raggiungere?

Grazie

**Comment by BS240872** [comment]
Buonasera se io cerco imprese che hanno vinto gare d’appalto in Campania in ambito PNNR  non c’è mai un file che mette in relazione nome impresa, cig e regione.
Ciao Gennaro a.

Da: Andrea Borruso ***@***.***>
Inviato: lunedì 7 ottobre 2024 17:28
A: ondata/italian-public-sector-pnrr-data-guide ***@***.***>
Cc: Gennaro Aversano ***@***.***>; Mention ***@***.***>
Oggetto: Re: [ondata/italian-public-sector-pnrr-data-guide] Ricerca Nominativi (Discussion #28)

!! WARNING External E-mail !! L’origine di questa e-mail è esterna al Gruppo Banca Sistema.
In caso di messaggi sospetti, NON cliccare sui link, NON aprire file allegati e NON rispondere o inoltrare l’e-mail, contatta l'Help Desk.


Non riesco ad acquisire esiti di gara per nominativo e per localizzazione , nominativi e localizzazione non sono mai associati

Ciao @BS240872<https://github.com/BS240872> scusaci, ma il tuo messaggio ci era sfuggito. È passato molto tempo: hai risolto?

Se non hai risolto, mi fai un esempio dell'obiettivo che hai e che non riesci a raggiungere?

Grazie
